fiftyone/core/config.py

```diff
@@ -198,6 +198,30 @@ def _validate(self):
             self.default_ml_backend = self.default_ml_backend.lower()
 
 
+class AnnotationConfig(EnvConfig):
```

Comment:
Hmmm, this config will grow as more providers as added? Not terribly important now, but maybe each provider should have a separate config.

Reply:
yeah, I agree. Still pondering how to best implement this, considering that I'd like for users to be able to write their own annotation plugin (supporting exact same syntax as natively supported backends) without editing code within the FiftyOne package.